### ISSUE TITLE
Natural sorting for the files

### ISSUE THREAD

**Issue by iRomanyshyn** [webtorrent/webtorrent-desktop#1884]
Please, implement natural sorting for the file names, because now files are sorted alphabetically only and it makes difficult to view some training videos with more than 10 series named and ordered like this.
```
Unit 1
Unit 10
Unit 11
...
Unit 18
Unit 19
Unit 2
Unit 20
Unit 21
```

**Comment by feross** [comment]
Agree that this would be useful

**Comment by alxhotel** [comment]
I believe this natural sorting has already been implemented in #1582.

https://github.com/webtorrent/webtorrent-desktop/blob/a20f82cc618cab9d42d02b255036dbed9db45536/src/renderer/pages/torrent-list-page.js#L288

**Comment by feross** [comment]
@alxhotel Nice!
